services/detectSingleShots.js

```diff
@@ -0,0 +1,45 @@
+function detectSingleShots(movementRatioList, facePredictionList) {
+  const movementRatioFrames = movementRatioList.map((obj) => {
+    return obj.movementRatio;
```

Comment:
민지님 obj보다 조금 더 명시적인 이름을 붙이시는 것은 어떠신가요?
ex) 
```suggestion
  const movementRatioFrames = movementRatioList.map((frameInfo) => {
    return frameInfo.movementRatio;
```

Reply:
민석님! 해당 정보는 프레임 움직임 비율에 대한 obj 값이기 때문에 `frameMovementInfo`으로 변경하겠습니다!👍